engine/src/main/scala/cromwell/engine/backend/BackendConfiguration.scala

```diff
@@ -28,12 +29,13 @@ object BackendConfiguration {
     BackendConfigurationEntry(
       backendName,
       entry.getString("actor-factory"),
-      entry.getConfig("config")
+      entry.getConfigOr("config")
     )
   }
 
   val DefaultBackendEntry: BackendConfigurationEntry = AllBackendEntries.find(_.name == DefaultBackendName) getOrElse {
-    throw new IllegalArgumentException(s"Could not find specified default backend name '$DefaultBackendName'.")
+    throw new IllegalArgumentException(s"Could not find specified default backend name '$DefaultBackendName' " +
+      s"in '${AllBackendEntries.mkString("', '")}'.")
```

Comment:
I suspect github's syntax highlighter [had a stroke](http://stackoverflow.com/questions/30667675/what-does-a-red-text-background-mean-in-github-comparison-view), possibly due to all the quoting: `"'"','"'"`